src/index.ts

```diff
@@ -97,9 +120,39 @@ export async function createDirigeraClient({
             },
           })
           .json()
+          .catch((err) => ({ access_token: undefined }))
```

Comment:
The `err` parameter is unused and `tsc` throws an error.

```suggestion
          .catch(() => ({ access_token: undefined }))
```